contributing/commit-messages.md

```diff
@@ -0,0 +1,122 @@
+[&larr; Table of Contents](../CONTRIBUTING.md)
+
+# Commits
+
+FDS uses conventional commits to produce release versions according to semantic release.
+
+Your commit messages must have a shape like
+
+```
+<type>[optional scope]: <description>
+
+[optional body]
+
+[optional footer(s)]
+```
+
+where `<type>` can be one of the following:
+
+- fix
+- feat
+- build
+- chore
+- ci
+- docs
+- style
+- refactor
+- perf
+- test
+
+NOTE: you can make use of `yarn commit` script to help you create a commit message that follows these guidelines.
+
+# How to produce a new version release
+
+- fix: a commit of the type fix patches a bug in your codebase (this correlates with PATCH in Semantic Versioning).
+- feat: a commit of the type feat introduces a new feature to the codebase (this correlates with MINOR in Semantic Versioning).
+- BREAKING CHANGE: a commit that has a footer BREAKING CHANGE:, or appends a ! after the type/scope, introduces a breaking API change (correlating with MAJOR in Semantic Versioning). A BREAKING CHANGE can be part of commits of any type.
+- The rest of the types don't produce a version release when they don't contain `BREAKING CHANGE` in the description/footer or `!` in the type.
+
+# Examples
+
+## Commit message with description and breaking change footer
```

Comment:
Breaking changes don't happen as often as patches and minors, so we should maybe start with examples of those first?

Reply:
makes sense, let me reorder it